wormhole-connect/src/utils/localStorage.ts

```diff
@@ -1,56 +1,71 @@
 import { TransactionLocal } from 'config/types';
 
-// Adds an object to localStorage
-export const addItemToLocalStorage = (
-  id: string, // Key for the item
-  data: object, // Item data
-  prefix: string, // Prefix to use for the item key
-  max?: number, // Max number of items allowed with the same prefix
-) => {
+const LOCAL_STORAGE_KEY = 'wormhole-connect:transactions:inprogress';
```

Comment:
seems like either this file or this const should have a more specific name... this isnt the only feature Connect uses `localStorage` for. I'd personally just rename this file to `txHistoryCache.ts` or something like that.

https://github.com/wormhole-foundation/wormhole-connect/blob/development/wormhole-connect/src/utils/wallet/index.ts#L132-L134

Reply:
Initially it had more generic functions but now very tx history specific. I'll rename the file 👍